src/Microsoft.Framework.Configuration.Json/JsonConfigurationSource.cs

```diff
@@ -78,7 +80,20 @@ public override void Load()
         internal void Load(Stream stream)
         {
             JsonConfigurationFileParser parser = new JsonConfigurationFileParser();
-            Data = parser.Parse(stream);
+            try
+            {
+                Data = parser.Parse(stream);
+            }
+            catch(Exception e)
+            {
+                var splitErrorMessageArray = e.Message.Split(
+                    new string[] { _exceptionMessageSplitString },
```

Comment:
This will fail if Newtonsoft.Json gets localized.  Catch the JsonReaderException and use `.LineNumber` directly. Then maybe you can use `.Path` to read the broken line and show it.

http://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_JsonReaderException.htm